brave/browser-laptop
# Using publisher data file for publisher exclusion (removing ruleset code)

Fixes: #14498

This removes the code related to caching/using rulesets and its related dependencies)

Depends on https://github.com/brave-intl/bat-client/pull/88/ (will need `bat-client 3.4.0` when available)

## Submitter Checklist:


## Test Plan:
(Code testing: Be sure to pull `bat-client` from https://github.com/brave-intl/bat-client/pull/88/  and re-install modules before testing)
1. Run Brave from this branch with a clean profile, enable payments
2. Ensure that auto-include is on
3. Pull up `publisher-data.json` for testing data (Note, file is a bit large)
4. Visit sites and satisfy the minimum requirements to add them to the ledger. (for example amazon.com)
5. Ensure that ledger table inclusion matches what is in `publisher-data.json`

## Reviewer Checklist:
